helm-chart/templates/service-monitor.yaml

```diff
@@ -0,0 +1,23 @@
+{{ if .Values.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "promscale.fullname" . }}
+  namespace: {{ template "promscale.namespace" . }}
+  labels:
+    app: {{ template "promscale.fullname" . }}
+    chart: {{ template "promscale.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  endpoints:
+  - interval: 30s
+    port: metrics-port
```

Comment:
Shouldn't this be `.Values.service.metricPort`?

Reply:
Because ServiceMonitor uses a `name` not port value. https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#endpoint